### DISCUSSION TITLE
是否加入每次获取最优IP解析到指定CF域名？

### DISCUSSION THREAD

**Discussion by murazi** [XIU2/CloudflareSpeedTest#449] [讨论]
### 功能需求

RT
比如每次手动获取优选/定时任务获取优选，就解析到cloudFlare下的指定域名，这样就可以不用再去更新host

不过这样相当于多了一层通信，但是方便啊，当较多设备需要更新host时，就省事

### 预期目标

可以半自动或定时任务全自动

**Comment by zanzhz1101** [comment]
啥意思，自动化吗
